ovos_audio/service.py

```diff
@@ -354,7 +355,7 @@ def _get_tts_fallback(self):
         """Lazily initializes the fallback TTS if needed."""
         if not self.fallback_tts:
             config = Configuration()
-            engine = config.get('tts', {}).get("fallback_module", "mimic")
+            engine = config.get('tts', {}).get("fallback_module", "")
             cfg = {"tts": {"module": engine,
```

Comment:
can we add
```python
if not engine:
   return
```

just to avoid the call to `TTSFactory.create(cfg)` below

Reply:
Sure thing, but there only seems to be 2 calls to  [ _get_tts_fallback](https://github.com/LordIvanhoe/ovos-audio/blob/c68d7c09702f6d59aefdfa31d0902c563bbe7286/ovos_audio/service.py#L354) and they are both guarded by:
 
`          if self._tts_hash != self._fallback_tts_hash:
`

and the other one has:
```
        if not self._fallback_tts_hash or \
                self._fallback_tts_hash != config.get("fallback_module", ""):
```
which should prevent it to ever getting there (in our case, where fallback_tts is empty) ? Or no?

Reply:
we have projects that subclass this and might be calling other methods directly such as execute_fallback_tts, just being defensive :)